.github/workflows/publish-stable.yml

```diff
@@ -0,0 +1,149 @@
+name: Stable Check for Minor Update + Automated Publish
+
+on:
+  schedule:
+    - cron: '*/15 * * * *'
```

Comment:
Is it possible to run this more frequently?

Reply:
yes, it's possible. Since this script only really gets triggered once every 2 weeks, I've thought it'd be best to run it less frequently.

Reply:
I reduced it to every 10 minutes